packages/math-input/src/components/keypad/button-assets.tsx

```diff
@@ -15,9 +15,7 @@ import * as React from "react";
 // TODO: This should be an enumeration of all of the possible legal values
 type KeyId = string;
 
-const buttonAsset: React.FC<{
-    id: KeyId;
-}> = function ({id}): React.ReactElement {
+const buttonAsset = function ({id}: {id: KeyId}): React.ReactElement {
```

Comment:
**nit**: Shouldn't this be `ButtonAsset` if it's a React component?  Looks like we avoid this issue because it's a default export and we export it named as `ButtonAsset`.  This is a bit outside your change, but would you mind tweaking this one?

Reply:
Will do.